charts/perses/values.yaml

```diff
@@ -112,7 +126,7 @@ config:
     # -- file system configs
     file:
       folder: /etc/perses/storage
-      extension: yaml
+      extension: json
```

Comment:
For some weird reason I wasn't able to load datasources using yaml format.
Is it supposed to work?

Reply:
ah yeah it's supposed to work for every resources supported. Probably there is a bug somewhere :o

Reply:
If when I come back from parental leaving and this is still an issue I can take a look.

Reply:
I think it is fixed through the PR https://github.com/perses/perses/pull/1444 ( I didn't yet check)